src/types/directory.ts

```diff
@@ -45,3 +45,13 @@ export type ResourcePageData = Required<Omit<PageData, "type">> & {
 export interface ResourceRoomNameUpdateProps {
   newDirectoryName: string
 }
+
+export interface GetMediaFoldersDto {
+  directories: DirectoryData[]
+  files: MediaData[]
+  total: number
```

Comment:
nit could we use `totalFiles` or `totalItems`? just to distinguish this from total directories?

Reply:
this is changed downstream so that `total` only exists on 

```ts
export interface GetMediaFilesDto {
  files: MediaFilePathData[]
  total: number
}
```